### PR TITLE
separate styles

### LINKED ISSUE #22: Creare 2 file SCSS per le 2 UI pubbliche: PUBLIC e AUTH
- Usare la funzione di import di SCSS/SASS.. per separare le aree già individuate con i commenti.. in sotto file scss da riusare nei 2 scss principali..

### PR DESCRIPTION
**Pull request by LucaCasella**

### CODE REVIEW

**Comment by antonio8101** [comment]
auth.scss should have a content similar to public.scss

File: resources/sass/app.scss [hunk 1]
```diff
@@ -6,260 +6,5 @@ $secondary: $gray-700;
 $font-family-base: 'Kulim Park', sans-serif;
```
Comment (antonio8101) [inline resources/sass/app.scss]: take also font in font.scss and colors in colors.scss

**Review by antonio8101 — CHANGES_REQUESTED** [review]
finally We expect to have 3 different main scss files. One for the public, one for the auth and one for app (private-area).

**Comment by LucaCasella** [comment]
At the moment the auth styles are inline in the related views and components

**Comment by antonio8101** [comment]
> At the moment the auth styles are inline in the related views and components

Can We move the style of the auth area from the inline to the proper file?
I think in anycase We don't need app.scss .. What do you think?

**Comment by LucaCasella** [comment]
Yes it seemed like a good idea.
So in viteconfig.js directly reference the 3 main scss?

**Pull request merged**